run_tests.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+echo building js files
+npm run build
```

Comment:
mozno pridat nejake try a v pripade failu urobit nejaky log, ako sme sa bavili
alebo by to slo mozno jednoducho
`npm run build > fail.log`
a potom to vsetko streamovat do `fail.log` a potom by sa tam hodilo sa v tom kroku zastavit

Reply:
ked to chces do logu, tak miesto `run_tests.sh` spustis `run_tests.sh > report.log`

Reply:
nerozumiem ako to myslis s tym zastavovanim

Reply:
no to je tym ze nerozumiem uplne ako funguje terminal, normalne by clovek cakal ze ked mas nejaky ten skript subor tak ze to pojde postupne zhora dole a ked nejaky ten prikaz zkonci chybou tak sa cely skript skonci,
a ze sa to teda nebude snazit urobit dalsi ten prikaz aj ked v tom predchodzom kroku bola chyba